infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServerNode.java

```diff
@@ -14,23 +13,11 @@
 import io.netty.handler.codec.LengthFieldPrepender;
 import io.netty.handler.ssl.SslContext;
 import io.netty.handler.ssl.SslHandler;
-
-import java.util.ArrayList;
-import java.util.Map;
-import java.util.Optional;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.atomic.AtomicBoolean;
-import java.util.regex.Pattern;
-
-import javax.annotation.Nonnull;
-import javax.net.ssl.SSLEngine;
-import javax.net.ssl.SSLException;
-
 import lombok.Getter;
+import lombok.NonNull;
 import lombok.extern.slf4j.Slf4j;
-
+import org.corfudb.infrastructure.configuration.CorfuConfig;
```

Comment:
Unused import.

We really need to enable sonarqube, ever since @xnull broke it  PRs have been adding unused imports and reordering packages.

Reply:
Agree, but we have to sacrifice - running tests in parallel
Also, I believe it's much easier and more useful is to set up Spotbugs and PMD plugins in maven and enable them by default